tools/c7n_mailer/c7n_mailer/replay.py

```diff
@@ -18,11 +18,11 @@
 
 import boto3
 import jsonschema
+import yaml
```

Comment:
it seems to be the change to this file thats triggering the coverage err, as this module doesn't have a test.

Reply:
adding one will resolve.

Reply:
Thanks Kapil! Total cov is sitting at 62% so I'll try to get up to about 75 or 80. It will be another day  or so before I am able to add it all, so call it a work in progress :)

Reply:
I ended up getting it to 70% and will let it sit there for now. Might come back later to see how much higher I can push. Don't want to hold up those other cards for too long right now.